package-lock.json

```diff
@@ -20480,12 +20469,18 @@
         "@babel/runtime": "^7.3.1",
         "highlight.js": "~9.13.0",
         "lowlight": "~1.11.0",
-        "prismjs": "1.21.0",
+        "prismjs": "^1.8.4",
```

Comment:
Not sure why this has downgraded itself?

Reply:
Try npx npm-force-resolutions & npm install to see if it gets restored